python/ray/_raylet.pyx

```diff
@@ -4101,6 +4101,7 @@ cdef class CoreWorker:
                                          worker.current_session_and_job)
         else:
             return ray.actor.ActorHandle(language, actor_id,
+                                         {},  # method is_generator
```

Comment:
better to make these kwargs

Reply:
btw, are you talking about 

```
def __init__(self,
    method is_generator=None
):
```

something like this?

Reply:
yes, use kwargs instead of a long list of optional positionals with comments